src/components/AuthForm.js

```diff
@@ -188,14 +188,41 @@ const authSchemas = {
         password: {"ui:widget": "password"}
       }
     }
-  }
+  },
+  portier: {
+    schema: {
+      ...baseAuthSchema,
+      required: [...baseAuthSchema.required, "email"],
+      properties: {
+        ...baseAuthSchema.properties,
+        email: {
+          title: "Email address",
+          type: "string",
+          format: "email",
+        }
+      }
+    },
+    uiSchema: {
+      authType: {
+        ...baseUISchema.authType,
+        "ui:help": (
+          <span>
+            <b>Note:</b> The
+            <a href="https://github.com/Kinto/kinto-portier">{" kinto-portier "}</a>
```

Comment:
Just a silly way to ensure spaces wrap the link inner text. React by default don't add spaces between elements separated by text nodes, as in regular HTML. 

So yeah that's ugly, but that prevented me to waste 5 minutes of my life in a CSS stylesheet, which was awesome.